zerver/lib/integrations.py

```diff
@@ -94,6 +96,40 @@ def add_doc_context(self, context):
         # type: (Dict[Any, Any]) -> None
         self.doc_context = context
 
+class BotIntegration(Integration):
+    DEFAULT_LOGO_STATIC_PATH_PNG = 'static/images/bots/{name}.png'
+    DEFAULT_LOGO_STATIC_PATH_SVG = 'static/images/bots/{name}.svg'
```

Comment:
(For the short term, I think the Zulip logo is probably a fine default for bots that don't have a logo file specified when the integration is declared; obviously for the GitHub one, we want the GitHub logo).